pkg/graph/resolvers/mto_solution.go

```diff
@@ -0,0 +1,138 @@
+package resolvers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cms-enterprise/mint-app/pkg/authentication"
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/cms-enterprise/mint-app/pkg/storage"
+	"github.com/cms-enterprise/mint-app/pkg/storage/loaders"
+)
+
+// MTOSolutionGetByModelPlanIDLOADER implements resolver logic to get all MTO
+// solutions by a model plan ID using a data loader
+func MTOSolutionGetByModelPlanIDLOADER(
+	ctx context.Context,
+	modelPlanID uuid.UUID,
+) ([]*models.MTOCommonSolution, error) {
+
+	// Translate a nil key to UUID nil, as we need a primitive type for translating results later
+	return loaders.MTOCommonSolution.ByModelPlanID.Load(ctx, modelPlanID)
+}
+
+func MTOSolutionGetCommonSolutionByKeyLoader(
```

Comment:
Whatever method's calling this can probably just be replaced directly by a call to `MTOCommonSolutionGetByKeyLOADER`